### PR TITLE
fix: vertically center 3+ letter abbreviations (#4179)

### PR DESCRIPTION
**Pull request by jsoref**
Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Optional. My organization is added to USERS.md.
* [x] I've signed the CLA and my build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

### CODE REVIEW

File: ui/src/app/applications/components/resource-icon.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const ResourceIcon = ({kind}: {kind: string}) => {
     const style: React.CSSProperties = {
         display: 'inline-block',
         verticalAlign: 'middle',
-        padding: '2px 4px',
+        padding: `${n <= 2 ? 2 : 0}px 4px`,
```
Comment (alexmt) [inline ui/src/app/applications/components/resource-icon.tsx:17]: I think it should be `2px ${n <= 2 ? 2 : 0}px`, is not it? Hope I'm not missing anything.

**Review by alexmt — CHANGES_REQUESTED** [review]

File: ui/src/app/applications/components/resource-icon.tsx (hunk 1 above)
Comment (jsoref) [reply ui/src/app/applications/components/resource-icon.tsx:17]: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

```css
padding: 10% 0;
```
image

I'm trying to change top-bottom alignment, which is the first unit.

Comment (alexmt) [reply ui/src/app/applications/components/resource-icon.tsx:17]: Ah, got it. I was looking at the wrong element. Thank you!

**Review by alexmt — APPROVED** [review]
LGTM

**Pull request merged**